src/Utilities/Compiler/Extensions/IOperationExtensions.cs

```diff
@@ -410,6 +410,24 @@ public static IOperation GetRoot(this IOperation operation)
             return operation;
         }
 
+        /// <summary>
+        /// Sorta like <see cref="GetRoot(IOperation)"/>, but not the root root.
```

Comment:
Needs more detailed explanation, with probably a simple example?

Reply:
Now I wonder if I even need this, given that `PropertySetAnalysis.BatchGetOrComputeHazardousUsages()` is looking through all local / anonymous functions in a CFG anyway.  I could probably just use `GetRoot()`.

Reply:
Nope, don't need it.  Will remove.